scripts/install_chrome_for_ci.py

```diff
@@ -28,7 +28,7 @@
 from scripts import common
 
 
-CHROME_VERSION = '88.0.4324.96-1'
+CHROME_VERSION = '89.0.4389.90-1'
```

Comment:
In the previous version, cookies were not getting added. Upgrading it to the next version fixes this.